MapboxNavigation/RouteMapViewController.swift

```diff
@@ -758,6 +827,63 @@ extension RouteMapViewController: BottomBannerViewDelegate {
     }
 }
 
+//MARK: - Keyboard Handling
+
+extension RouteMapViewController {
+    fileprivate func subscribeToKeyboardNotifications() {
```

Comment:
IMHO, it's easier to keep track of these notifications if they're all in `resumeNotifications/suspendNotifications` but feel free to leave it as it is.